src/transformers/models/clip/modeling_clip.py

```diff
@@ -68,7 +68,8 @@ def contrastive_loss(logits: torch.Tensor) -> torch.Tensor:
 
 def clip_loss(similarity: torch.Tensor) -> torch.Tensor:
     caption_loss = contrastive_loss(similarity)
-    image_loss = contrastive_loss(similarity.T)
+    # .T doesn't work while converting to onnx, aten::numpy_T operator is not supported yet
```

Comment:
```suggestion
```
I'd remove this comment in the code as there is nothing surprising to the user when they see a `.t()` (they wouldn't expect a `.T` in its place).